lib/uncruft/deprecate_attribute.rb

```diff
@@ -0,0 +1,30 @@
+module Uncruft
+  module DeprecateAttribute
+    extend ActiveSupport::Concern
+
+    module ClassMethods
+      def deprecate_attribute(attribute, aliased_attribute: nil, message:)
```

Comment:
This could be considered a v2 feature since I believe the majority of potential use cases we've seen have been AR attributes, but I could see a `deprecate_method` that doesn't assume there's both a reader and writer being useful.

Reply:
Agreed. Not necessary as part of this initial release, but I could see `deprecate_attribute` being a convenience wrapper for the combination `deprecate_reader` and `deprecate_writer`.

Reply:
I'm skeptical `reader` and `writer` could exclusively cover all cases for this. I could see us deprecating methods that don't fit into that framing (e.g. a balance method that takes a date as an argument).

Reply:
Fair enough! There's probably a generic "this method is deprecated" method (regardless of # of args or whether it is a reader/writer) that is the superset of all of the above.